functorch/_src/partitioners.py

```diff
@@ -246,6 +247,24 @@ def _count_ops(graph):
     print(sorted(cnt.items(), key=lambda x: x[1], reverse=True))
 
 
+@functools.lru_cache(None)
+def pointwise_ops():
+    ops = []
+    for attr_name in dir(torch.ops.aten):
+        opoverloadpacket = getattr(torch.ops.aten, attr_name)
+        if not isinstance(opoverloadpacket, torch._ops.OpOverloadPacket):
+            continue
+
+        for overload in opoverloadpacket.overloads():
+            op_overload = getattr(opoverloadpacket, overload)
+            if torch.Tag.pointwise in op_overload.tags:
+                # currently aot autograd uses packet not overload
+                ops.append(opoverloadpacket)
+                break
+
+    return ops
```

Comment:
If at all possible, we should figure out a way to do this that doesn't involve glomming all of the operators into an LRU cache. The main hazard to doing it this way is that torch.ops.aten is lazily loaded, so there isn't a guarantee that you will actually have all of the operators at the time you call this function. And this will be even worse if you want to support pointwise ops outside of the aten namespace

Reply:
yeah that's a good idea. I think we can just create it in codegen when the native functions are parsed. 

Currently, there's no API for users to add tags from Python Library API but in the future, we'll have to ensure that this db is up-to-date when new ops are added.

Reply:
I mentioned this in WC but to repeat it here: for use in the partitioner, you don't need the entire set, at the point where you need to test if an op is pointwise or not, just check the tag directly.